### DISCUSSION TITLE
How do you get the nuget package number to not be 1.0.0 ?

### DISCUSSION THREAD

**Discussion by JohnGoldInc** [OData/odata.net#2654] [Q&A]
I change the Assembly/File versions in the properties of projects to 8.0.0.0 but still get ...1.0.0.nupkg

And I don't see anything controlling it in build.ps1 or build.cmd regarding generating packages...

I am sure it's in tools somewhere, as I see something in CustomMSBuild that looks like it's doing something with packaging and versioning, but I need to figure out how to invoke it correctly.

**Comment by habbes** [comment]
Hi @JohnGoldInc does this question relate to the OData libraries or to .NET libraries in general? For which library do you want to update the version number?

**Reply by JohnGoldInc** [reply]
the nuget pack version numbers of the projects referenced in OData.sln,

 I just want to increment them so I can test changes to code locally, and for some reason, they are ignoring the numbers set in Project Properties and staying 1.0.0 when I run the VS pack command.
 
 And I can't leave them as 1.0.0 because I am using them with other libraries that have minimum version requirements. (ie. >7.15)

**Reply by habbes** [reply]
In this repo, we set the version number in the file `tools/CustomMSBuild/Versioning.props` in the following XML properties:

```xml
<!-- Set the version number: major, minor, build and release (i.e. alpha, beta or blank for RTM)-->
  <PropertyGroup>
    <VersionMajor Condition="'$(VersionMajor)' == ''">7</VersionMajor>
    <VersionMinor Condition="'$(VersionMinor)' == ''">15</VersionMinor>
    <VersionBuildNumber Condition="'$(VersionBuildNumber)' == ''">0</VersionBuildNumber>
    <VersionRelease Condition="'$(VersionRelease)' == ''"></VersionRelease>
  </PropertyGroup>
```
Then the rest is handled by the build process.

**Comment by JohnGoldInc** [comment]
And what is the command to pack nuget packages? Build.cmd didn’t seem to
make packages (and Visual Studio Pack makes them 1.0.0)
